libr/core/project.c

```diff
@@ -630,7 +630,10 @@ static bool simpleProjectSaveScript(RCore *core, const char *file, int opts) {
 
 static bool projectSaveScript(RCore *core, const char *file, int opts) {
 	char *filename, *hl, *ohl = NULL;
+	char *binpath = r_file_abspath(core->bin->file);
+	char  *reopen = (char *) r_malloc(strlen(binpath) + 33);
 	int fd, fdold;
+	sprintf(reopen, "\"o %s\"\n", binpath);
```

Comment:
just use r_str_newf() and avoid r_malloc+sprintf

Reply:
also, can you describe which is the exact issue and what do this solves? the idea of projects is that they are tied to one specific file which must be copied into the projects directory imho.

the problem is that bininfo will not be loaded otherwise.

maybe we can have an o command that loads the file the bininfo, but doesnt sets any flag from them. so the projects may be able to load and map all the files and then just reload the flags from the previous session and keep consistency